### PR TITLE
docker fix with ajv and ifconfig

### PR DESCRIPTION
**Pull request by sw1pr0g**

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
+    "build": "REACT_APP_HOST=$(ip addr show eth0 | grep 'inet ' | awk '{print $2}' | cut -d/ -f1) react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
   "resolutions": {
     "react-error-overlay": "6.0.9"
   },
   "dependencies": {
+    "ajv": "^6.12.6",
+    "ajv-keywords": "^3.5.2",
```
Comment (Kropanov) [inline web/package.json:17]: Why did you add this to deps?

File: web/package.json [hunk 2]
```diff
@@ -4,15 +4,17 @@
   "private": true,
   "scripts": {
     "preinstall": "npx npm-force-resolutions",
-    "start": "REACT_APP_HOST=$(ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\\.){3}[0-9]*).*/\\2/p') react-scripts start",
-    "build": "REACT_APP_HOST=$(ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\\.){3}[0-9]*).*/\\2/p') react-scripts build",
+    "start": "REACT_APP_HOST=$(ip addr show eth0 | grep 'inet ' | awk '{print $2}' | cut -d/ -f1) react-scripts start",
+    "build": "REACT_APP_HOST=$(ip addr show eth0 | grep 'inet ' | awk '{print $2}' | cut -d/ -f1) react-scripts build",
```
Comment (Kropanov) [inline web/package.json:8]: What is reason to change it?

File: web/.env [hunk 3]
```diff
@@ -1,3 +1,3 @@
 GENERATE_SOURCEMAP=false
-REACT_APP_ENVIRONMENT=production
+REACT_APP_ENVIRONMENT=development
```
Comment (Kropanov) [inline web/.env:2]: Why it's not production?

File: web/package.json (hunk 2 above)
Comment (sw1pr0g) [reply web/package.json:8]: ifconfig not accesible on most of debian releases what use ubuntu

**Review by Kropanov — APPROVED** [review]
